deflate.h

```diff
@@ -251,18 +252,22 @@ typedef struct internal_state {
     unsigned int matches;         /* number of string matches in current block */
     unsigned int insert;          /* bytes at end of window left to insert */
 
-#ifdef ZLIB_DEBUG
+    /* compressed_len and bits_sent are only used if ZLIB_DEBUG is defined */
     unsigned long compressed_len; /* total bit length of compressed file mod 2^32 */
     unsigned long bits_sent;      /* bit length of compressed data sent mod 2^32 */
-#endif
+
+    /* Reserved for future use */
```

Comment:
Maybe just add a comment to this to indicate it is here for alignment purposes? `Reserved for future use and alignment`. And same with the comment one below it.

Reply:
Fixed.